src/Host.cpp

```diff
@@ -1225,3 +1225,57 @@ void Host::refreshPackageFonts()
         installPackageFonts(package);
     }
 }
+
+void Host::setWideAmbiguousEAsianGlyphs(const Qt::CheckState state)
```

Comment:
I'm not sure how this crept in here - though I *was* a bit puzzled why you merged in a branch from my repo - so I guess that caused this... I guess it will come out in the ~~wash~~merge. 😖